src/notebook/Notebook.jl

```diff
@@ -54,10 +54,26 @@ Base.@kwdef mutable struct Notebook
     wants_to_interrupt::Bool=false
     last_save_time::typeof(time())=time()
     last_hot_reload_time::typeof(time())=zero(time())
+    last_serialized_version::String = ""
```

Comment:
Let's track this in pluto-vscode instead

Reply:
The benefits of having this here too are
- we prevent multiple file IO
- we prevent emitting events for file change (through listeners) if the file wasn't changed (that's a weaker argument - it can be part of the listener)
- we can compare this version to the `last_unsaved_serialized_version` (incoming) that is what we want `Shift+Enter` to propagate to the VS Code workspace, before `Control + S` actually does the filesave(!) (can also be tracked in the listener)

Reply:
all in all, I like it here for the above reasons - but we don't _really_ need it
What about having a hash of the (string contents of the) file here?